<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PLSErrorCodeTransformTool</class>
 <widget class="QWidget" name="PLSErrorCodeTransformTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1">
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,0">
     <item row="4" column="2">
      <widget class="QPushButton" name="pushButton_start">
       <property name="text">
        <string>Start Transform</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>output json</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="placeholderText">
        <string notr="true">eg:  /Users/jimbo/Downloads/PRISMLiveStudio_ErrorCode (39).xlsx</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPlainTextEdit" name="plainTextEdit_2">
       <property name="placeholderText">
        <string notr="true">eg:  /Users/jimbo/Downloads/PRISMLiveStudio_ErrorCode (39).xlsx</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButton_file">
       <property name="text">
        <string>select file</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>input excel</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>select file</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="pushButton_default">
       <property name="text">
        <string>Set Default Ouput</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item alignment="Qt::AlignLeft">
    <widget class="QPushButton" name="pushButton_clear">
     <property name="text">
      <string>clear logs</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
